deeprvat/data/dense_gt.py

```diff
@@ -475,6 +560,9 @@ def setup_annotations(
                 annotation_file, columns=list(set(columns)), engine="pyarrow"
             ).compute()
             self.annotation_df = self.annotation_df.set_index("id")
+            self.gene_specific_anno = self.annotation_df["gene_id"].dtype != np.dtype(
```

Comment:
This is duplicated. Won't break anything, but it can be deleted

Reply:
updated this